src/app/3d/qgs3dmapconfigwidget.cpp

```diff
@@ -492,3 +506,71 @@ void Qgs3DMapConfigWidget::validate()
   emit isValidChanged( valid );
 }
 
+void Qgs3DMapConfigWidget::init3dAxisPage()
```

Comment:
This function is unnecessary. it is better to create a slot in Qgs3DMapConfigWidget and connect UI signals to it

Reply:
The constructor is a bit overloaded, I wanted to avoid to add other parameter initializations. How can it be useful as a slot?